packages/storefront-lwc/src/modules/commerce/basket/basket.html

```diff
@@ -176,7 +176,7 @@ <h2 class="number-of-items">
 
                     <commerce-shippingmethods shipping-label="Shipping" shipping-methods={shippingMethods} selected-shipping-method-id={selectedShippingMethodId}></commerce-shippingmethods>
 
-                    <commerce-carttotals></commerce-carttotals>
+                    <commerce-basket-totals></commerce-basket-totals>
```

Comment:
Should it be `<commerce-baskettotals></commerce-baskettotals>` ?

Reply:
intentionally switched because "basket totals" is two separate words, not a compound word

Reply:
![image](https://user-images.githubusercontent.com/1566869/76315952-14be3f80-62b0-11ea-9b61-d46b37ca881e.png)

Reply:
It is camel-cased now